src/qz/ui/AboutDialog.java

```diff
@@ -1,150 +1,158 @@
 import qz.ui.component.LinkLabel;
-import qz.utils.SystemUtilities;
 
 import javax.swing.*;
-import javax.swing.border.EmptyBorder;
-import javax.swing.border.EtchedBorder;
 import java.awt.*;
-import java.net.MalformedURLException;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.font.TextAttribute;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
 import java.net.URL;
+import java.util.HashMap;
 import java.util.Map;
-import java.util.SortedMap;
 
 /**
  * Created by Tres on 2/26/2015.
  * Displays a basic about dialog
  */
 public class AboutDialog extends BasicDialog implements Themeable {
-    JPanel gridPanel;
-    JLabel wssLabel;
-    JLabel wsLabel;
-    SortedMap<String, String> libVersions;
-    String name;
 
-    public AboutDialog(JMenuItem menuItem, IconCache iconCache, String name) {
-        super(menuItem, iconCache);
-        this.name = name;
-        initComponents();
-    }
-
-    @Override
-    public void setVisible(boolean b) {
-        if (libVersions == null) {
-            libVersions = SecurityInfo.getLibVersions();
-            GridLayout layout = (GridLayout)gridPanel.getLayout();
-            layout.setRows(layout.getRows() + libVersions.size());
-
-            gridPanel.add(createLabel("Library Name:", true));
-            gridPanel.add(createLabel("Version:", true));
-            for (Map.Entry<String, String> entry: libVersions.entrySet()) {
-                gridPanel.add(createLabel("    " + entry.getKey(), false));
-                gridPanel.add(createLabel("    "  + (entry.getValue() == null ? "(unknown)" : entry.getValue()), false));
-            }
-            shadeComponents();
-        }
-        super.setVisible(b);
-    }
+    private static final Logger log = LoggerFactory.getLogger(AboutDialog.class);
 
-    public void initComponents() {
-        JComponent header = setHeader(new JLabel(getIcon(IconCache.Icon.BANNER_ICON)));
-        header.setBorder(new EmptyBorder(Constants.BORDER_PADDING, 0, Constants.BORDER_PADDING, 0));
-
-        gridPanel = new JPanel();
+    private Server server;
+    private Version latestVersion;
 
-        JScrollPane pane = new JScrollPane(gridPanel);
-        pane.getVerticalScrollBar().setUnitIncrement(8);
-        pane.setPreferredSize(new Dimension(0, (int)(100 * SystemUtilities.getWindowScaleFactor())));
-        gridPanel.setLayout(new GridLayout(5, 2));
-        gridPanel.setBorder(new EtchedBorder(EtchedBorder.LOWERED));
 
-        wsLabel = new JLabel("None");
-        wssLabel = new JLabel("None.  No https support.");
-        wssLabel.setFont(wsLabel.getFont().deriveFont(Font.BOLD));
+    public AboutDialog(JMenuItem menuItem, IconCache iconCache) {
+        super(menuItem, iconCache);
 
-        gridPanel.add(createLabel("Software:", true));
-        gridPanel.add(createLabel(name));
+        try {
+            URL api = new URL("https://api.github.com/repos/qzind/tray/releases");
```

Comment:
This should use the value from `Constants.java`.  Also, if it's blank, we need to disable this functionality (e.g. white-label builds).